src/utils/isValidInfo.ts

```diff
@@ -11,7 +11,9 @@ const isValidUrl = (url: string) => {
   try {
     const urlObj = new URL(url);
 
-    const urlRegex = new RegExp(/http(s?):\/\/(?:www\.)?[\w-]+\.[a-zA-Z]{2,}/);
+    const urlRegex = new RegExp(
+      /^https?:\/\/(?:www\.)?(?:[a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}(\/[^\s?#]*)?(?:\?[^\s#]*)?(?:#[^\s]*)?$/,
+    );
```

Comment:
Ik snap niet helemaal waarom je nog een regex test doet, als je new URL() doet, dat is al streng als het om validatie gaat.

Als je toch de regex wilt houden zou ik deze als constante bovenaan zetten, samen met

const EMAIL_REGEX = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,})+$/;
const PHONE_REGEX = /^[0-9\/\-_.+ ]{0,15}$/;